pkg/logs/config/integration_config.go

```diff
@@ -8,14 +8,17 @@ package config
 import (
 	"fmt"
 	"regexp"
+
+	"github.com/DataDog/datadog-agent/pkg/util/containers"
 )
 
 // Logs source types
 const (
 	TCPType          = "tcp"
 	UDPType          = "udp"
 	FileType         = "file"
-	DockerType       = "docker"
+	DockerType       = containers.RuntimeNameDocker
+	ContainerdType   = containers.RuntimeNameContainerd
```

Comment:
There is no config for `containerd` yet and I don't think we'll need one since we'd like to deprecate `docker`.